src/components/ui/CatCard.tsx

```diff
@@ -5,19 +5,31 @@ import { get } from "lodash";
 import { Staff } from "@/types/common-types";
 
 type Props = {
-  cat: Staff | any;
+  cat?: Staff | any;
+  catDeal?: Staff | any;

```

Comment:
remove catDeal here, this card only need cat object

Reply:
fixed